go.mod

```diff
@@ -7,77 +7,80 @@ require (
 	github.com/ghodss/yaml v1.0.1-0.20220118164431-d8423dcdf344
```

Comment:
side note: should we update the module name to be? 

`module github.com/mesosphere/kubefed`

Reply:
i see no harm in doing that (apart from all the refactoring thats needed in kommander repo) but also, there is no real upside (aesthetics aside). we want to get rid of kubefed eventually anyway so maybe _this is fine_ :see_no_evil: